packages/medusa/src/services/price-list.ts

```diff
@@ -255,7 +261,11 @@ class PriceListService extends BaseService {
     const priceListRepo = this.manager_.getCustomRepository(this.priceListRepo_)
```

Comment:
same as above, all public methods provided by a service should be wrapped in an `atomicPhase_` that way, if the consumer call a public method without using withTransaction, the entire chain related to that method will be wrapped in a transaction